zebra-chain/src/sapling/output.rs

```diff
@@ -134,12 +134,45 @@ impl ZcashSerialize for OutputInTransactionV4 {
 
 impl ZcashDeserialize for OutputInTransactionV4 {
     fn zcash_deserialize<R: io::Read>(mut reader: R) -> Result<Self, SerializationError> {
+        // # Consensus
+        //
+        // > Elements of a Output description MUST be valid encodings of the types given above.
+        //
+        // https://zips.z.cash/protocol/protocol.pdf#outputdesc
```

Comment:
Good point. The template we decided did not have the angle brackets, but in hindsight we should have those.

Since it seems too late to change this now, my suggestion is wait for https://github.com/ZcashFoundation/zebra/issues/2892 . `cargo doc` warns if you use a bare URL. (It will not complain for normal comments, but I think that's OK. If we ever turn a normal comment into a doc comment it will complain again)